internal/handler/site/site_handler.go

```diff
@@ -49,6 +49,7 @@ func (h *SiteHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		}
 		h.next.ServeHTTP(w, r)
 
+    w.Header().Set("Cache-Control", "public, max-age=31536000, no-cache")
```

Comment:
This is ineffective; `h.next` is a chain of middleware, ending with `serveFile`, which writes the static file to response. After writing the static file, headers can no longer be set, so this call would be ignored.